### ISSUE TITLE
loss is about 9？？How to make loss drop 

### ISSUE THREAD

**Issue by taoyunuo** [wy1iu/sphereface#59]
my solver ：
base_lr: 0.01
lr_policy: "multistep"
gamma: 0.1

stepvalue: 16000
stepvalue: 24000
stepvalue: 28000
max_iter: 28000

display: 100
momentum: 0.9
weight_decay: 0.0005
#snapshot: 1000
snapshot_prefix: "result/sphereface_model_old"

**Comment by taoyunuo** [comment]
I1130 03:26:36.153867 29311 sgd_solver.cpp:106] Iteration 4100, lr = 0.01
I1130 03:28:35.267403 29311 solver.cpp:228] Iteration 4200, loss = 18.6884
I1130 03:28:35.267515 29311 solver.cpp:244]     Train net output #0: lambda1 = 5
I1130 03:28:35.267521 29311 solver.cpp:244]     Train net output #1: lambda2 = 5
I1130 03:28:35.267529 29311 solver.cpp:244]     Train net output #2: softmax_loss1 = 9.34578 (* 1 = 9.34578 loss)
I1130 03:28:35.267532 29311 solver.cpp:244]     Train net output #3: softmax_loss2 = 9.34257 (* 1 = 9.34257 loss)
I1130 03:28:35.267536 29311 sgd_solver.cpp:106] Iteration 4200, lr = 0.01
I1130 03:30:34.356006 29311 solver.cpp:228] Iteration 4300, loss = 18.6907
I1130 03:30:34.356077 29311 solver.cpp:244]     Train net output #0: lambda1 = 5
I1130 03:30:34.356083 29311 solver.cpp:244]     Train net output #1: lambda2 = 5
I1130 03:30:34.356089 29311 solver.cpp:244]     Train net output #2: softmax_loss1 = 9.3441 (* 1 = 9.3441 loss)
I1130 03:30:34.356094 29311 solver.cpp:244]     Train net output #3: softmax_loss2 = 9.34659 (* 1 = 9.34659 loss)
I1130 03:30:34.356098 29311 sgd_solver.cpp:106] Iteration 4300, lr = 0.01

**Comment by zjz5250** [comment]
i  meet the same problem，did you sovle it，and how

**Comment by honghuCode** [comment]
change the parameter of lambda_min = 10 or bigger,the network can coverage

**Comment by aikudexiaohai** [comment]
@honghuCode how much is the loss and accuracy after you train

**Comment by honghuCode** [comment]
@aikudexiaohai  about 3

**Comment by aikudexiaohai** [comment]
@honghuCode thank you. and what about the accuracy?
